@xen-orchestra/lite/src/components/host/system/HostSystemHardwareSpecifications.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <UiCard>
+    <UiTitle>
+      {{ $t('hardware-specifications') }}
+    </UiTitle>
+    <VtsQuickInfoRow
+      :label="$t('manufacturer-info')"
+      :value="`${host.bios_strings['system-manufacturer']} (${host.bios_strings['system-product-name']})`"
+    />
+    <VtsQuickInfoRow
+      :label="$t('bios-info')"
+      :value="`${host.bios_strings['bios-vendor']} (${host.bios_strings['bios-version']})`"
+    />
+    <VtsQuickInfoRow :label="$t('cpu-model')" :value="host.cpu_info.modelname" />
+    <VtsQuickInfoRow :label="$t('core-socket')" :value="`${host.cpu_info.cpu_count} (${host.cpu_info.socket_count})`" />
+    <VtsQuickInfoRow disabled :label="$t('hyper-threading')" />
+    <VtsQuickInfoRow :label="$t('gpus')">
+      <template #value>
+        <template v-if="pGpus.length > 0">
+          {{ pGpusIds }}
+        </template>
```

Comment:
```suggestion
     <template v-if="pciNameDevice">
          {{ pciNameDevice }}
        </template>
```

You can change the name of the const if isn't good ^^